From bc98b254eea5d0323bf3eaf1ea97781bc3a9844f Mon Sep 17 00:00:00 2001
From: He Yong <hyyoxhk@163.com>
Date: Wed, 12 Apr 2023 23:59:44 +0800
Subject: [PATCH 4/4] Use CMAKE_INSTALL_FULL_LIBDIR instead of
 CMAKE_INSTALL_LIBDIR

Upstream-Status: Pending

Signed-off-by: He Yong <hyyoxhk@163.com>
---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 24f9dff7..fe85686a 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -160,7 +160,7 @@ endif()
 
 # directory needed by bsd
 if(NOT CMAKE_INSTALL_LIBDATADIR)
-  set(CMAKE_INSTALL_LIBDATADIR "${CMAKE_INSTALL_LIBDIR}" CACHE PATH "libdata root (LIBDIR)")
+  set(CMAKE_INSTALL_LIBDATADIR "${CMAKE_INSTALL_FULL_LIBDIR}" CACHE PATH "libdata root (LIBDIR)")
 endif()
 
 get_filename_component(CMAKE_INSTALL_FULL_LIBDATADIR "${CMAKE_INSTALL_LIBDATADIR}" ABSOLUTE)
-- 
2.34.1

