From f29469c49cf1de26bb9fca96d72ae4e77005000f Mon Sep 17 00:00:00 2001
From: Takuro Ashie <ashie@clear-code.com>
Date: Wed, 4 Aug 2021 14:09:58 +0900
Subject: [PATCH 1/3] Add CMAKE_SYSROOT to find additional cmake file

Upstream-Status: Inappropriate [embedded specific]

Signed-off-by: Takuro Ashie <ashie@clear-code.com>
---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 1268dd1..150ddf3 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -22,7 +22,7 @@ else()
   set(FMT_TARGET fmt::fmt)
 endif ()
 
-include("${FCITX_INSTALL_CMAKECONFIG_DIR}/Fcitx5Utils/Fcitx5CompilerSettings.cmake")
+include("${CMAKE_SYSROOT}${FCITX_INSTALL_CMAKECONFIG_DIR}/Fcitx5Utils/Fcitx5CompilerSettings.cmake")
 
 #######################################################################
 # Options
-- 
2.34.1

