From 4f659999dc7ad1a95a4056fb500d09c87f170dd9 Mon Sep 17 00:00:00 2001
From: Takuro Ashie <ashie@clear-code.com>
Date: Wed, 4 Aug 2021 14:11:13 +0900
Subject: [PATCH 2/3] Add cmake option ENABLE_METAINFO

Upstream-Status: Inappropriate [embedded specific]

Signed-off-by: Takuro Ashie <ashie@clear-code.com>
---
 CMakeLists.txt | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 2f829e4..0698723 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -94,9 +94,11 @@ if (ENABLE_TEST)
     endif()
 endif ()
 
+if (ENABLE_METAINFO)
 fcitx5_translate_desktop_file(org.fcitx.Fcitx5.Addon.ChineseAddons.metainfo.xml.in
     org.fcitx.Fcitx5.Addon.ChineseAddons.metainfo.xml XML)
 
 install(FILES "${CMAKE_CURRENT_BINARY_DIR}/org.fcitx.Fcitx5.Addon.ChineseAddons.metainfo.xml" DESTINATION ${CMAKE_INSTALL_DATADIR}/metainfo)
+endif()
 
 feature_summary(WHAT ALL FATAL_ON_MISSING_REQUIRED_PACKAGES)
-- 
2.34.1

