From 7fcbdfa5402cb227c36785c3d98b29bba4c0081f Mon Sep 17 00:00:00 2001
From: Takuro Ashie <ashie@clear-code.com>
Date: Wed, 4 Aug 2021 14:49:55 +0900
Subject: [PATCH 2/4] Add CMAKE_SYSROOT to find additional cmake file

Upstream-Status: Inappropriate [embedded specific]

Signed-off-by: Takuro Ashie <ashie@clear-code.com>
---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 21102aa..101a10a 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -32,7 +32,7 @@ find_package(PkgConfig REQUIRED)
 pkg_check_modules(ZSTD REQUIRED IMPORTED_TARGET "libzstd")
 
 find_package(Fcitx5Utils REQUIRED)
-include("${FCITX_INSTALL_CMAKECONFIG_DIR}/Fcitx5Utils/Fcitx5CompilerSettings.cmake")
+include("${CMAKE_SYSROOT}${FCITX_INSTALL_CMAKECONFIG_DIR}/Fcitx5Utils/Fcitx5CompilerSettings.cmake")
 
 find_package(Boost 1.61 REQUIRED COMPONENTS iostreams filesystem)
 set(LIBIME_INSTALL_PKGDATADIR "${CMAKE_INSTALL_FULL_DATADIR}/libime")
-- 
2.34.1

